src/main/java/core/basesyntax/figures/FigureSupplier.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.color.Color;
+import core.basesyntax.color.ColorSuplier;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random RANDOM = new Random();
+    private static final int FIGURES = 5;
```

Comment:
```suggestion
    private static final int FIGURES_QUANTITY = 5;
```
or 
```suggestion
    private static final int FIGURES_SIZE = 5;
```

Reply:
private Figure getFigure() {
        final int figures = 5;
        final int maxSize = 99;
        ColorSuplier colorSuplier = new ColorSuplier();

I've already fix it. Actual version is: 
public class FigureSupplier {
    private static final Random RANDOM = new Random();
    private static final int FIGURES = 5;
    private static final int MAX_SIZE = 99;
    
So...